apps/ftu/test/unit/language_test.js

```diff
@@ -55,19 +55,14 @@ suite('languages >', function() {
     document.body.appendChild(section);
     var list = document.createElement('ul');
     section.appendChild(list);
-
-    // XXX in reality this method is async b/c it uses LanguageList.get;  here
-    // however it uses the mock which is sync.  Fix this in bug 1119865.
-    LanguageManager.buildLanguageList();
```

Comment:
Last time I looked at this code, I remember thinking that `buildLanguageList` should take a callback so that we can know in the tests when it finishes.  Right now we work around this by using sync mocks inside of it, but once we switch to a Promise-based API for `LanguageList`, we'll need a way to notify the test's setup that the FTU has finished building the list.